Description: properly set the styleMapFamilyName fields
 so that e.g. Comic Neue Bold is recognized as the bold
 variant of Comic Neue instead of a separate font.
Author: Fabian Greffrath <fabian@debian.org>

--- a/UFO/ComicNeue-Angular-Regular.ufo/fontinfo.plist
+++ b/UFO/ComicNeue-Angular-Regular.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>Normal</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Angular Regular</string>
+    <string>Comic Neue Angular</string>
     <key>styleMapStyleName</key>
     <string>regular</string>
     <key>versionMajor</key>
--- a/UFO/ComicNeue-Angular_Bold.ufo/fontinfo.plist
+++ b/UFO/ComicNeue-Angular_Bold.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>Bold</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Angular Bold</string>
+    <string>Comic Neue Angular</string>
     <key>styleMapStyleName</key>
     <string>bold</string>
     <key>versionMajor</key>
--- a/UFO/ComicNeue-Angular_Bold_Oblique.ufo/fontinfo.plist
+++ b/UFO/ComicNeue-Angular_Bold_Oblique.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>BoldOblique</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Angular Bold Oblique</string>
+    <string>Comic Neue Angular</string>
     <key>styleMapStyleName</key>
     <string>bold italic</string>
     <key>versionMajor</key>
--- a/UFO/ComicNeue-Angular_Light_Oblique.ufo/fontinfo.plist
+++ b/UFO/ComicNeue-Angular_Light_Oblique.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>LightOblique</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Angular Light Oblique</string>
+    <string>Comic Neue Angular Light</string>
     <key>styleMapStyleName</key>
     <string>italic</string>
     <key>versionMajor</key>
--- a/UFO/ComicNeue-Angular_Oblique.ufo/fontinfo.plist
+++ b/UFO/ComicNeue-Angular_Oblique.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>Oblique</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Angular Oblique</string>
+    <string>Comic Neue Angular</string>
     <key>styleMapStyleName</key>
     <string>italic</string>
     <key>versionMajor</key>
--- a/UFO/ComicNeue-Regular.ufo/fontinfo.plist
+++ b/UFO/ComicNeue-Regular.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>Normal</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Regular</string>
+    <string>Comic Neue</string>
     <key>styleMapStyleName</key>
     <string>regular</string>
     <key>versionMajor</key>
--- a/UFO/ComicNeue_Bold.ufo/fontinfo.plist
+++ b/UFO/ComicNeue_Bold.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>Bold</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Bold</string>
+    <string>Comic Neue</string>
     <key>styleMapStyleName</key>
     <string>bold</string>
     <key>versionMajor</key>
--- a/UFO/ComicNeue_Bold_Oblique.ufo/fontinfo.plist
+++ b/UFO/ComicNeue_Bold_Oblique.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>BoldOblique</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Bold Oblique</string>
+    <string>Comic Neue</string>
     <key>styleMapStyleName</key>
     <string>bold italic</string>
     <key>versionMajor</key>
--- a/UFO/ComicNeue_Light_Oblique.ufo/fontinfo.plist
+++ b/UFO/ComicNeue_Light_Oblique.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>LightOblique</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Light Oblique</string>
+    <string>Comic Neue Light</string>
     <key>styleMapStyleName</key>
     <string>italic</string>
     <key>versionMajor</key>
--- a/UFO/ComicNeue_Oblique.ufo/fontinfo.plist
+++ b/UFO/ComicNeue_Oblique.ufo/fontinfo.plist
@@ -7,7 +7,7 @@
     <key>styleName</key>
     <string>Oblique</string>
     <key>styleMapFamilyName</key>
-    <string>Comic Neue Oblique</string>
+    <string>Comic Neue</string>
     <key>styleMapStyleName</key>
     <string>italic</string>
     <key>versionMajor</key>
